lib/recurly/client.php

```diff
@@ -24,7 +24,7 @@ class Recurly_Client
   /**
    * API Version
    */
-  public static $apiVersion = '2.29';
+  public static $apiVersion = '2.99';
```

Comment:
you can run against 574 branch and use 2.29

Reply:
ok changed :D